_posts/en/newsletters/2019-08-21-newsletter.md

```diff
@@ -0,0 +1,107 @@
+  into Bitcoin Core as the point where those forks activate.  This
+  means any block chain reorganization that extends further back than
+  those blocks can create a chainsplit between nodes with this
+  hardcoding and those without it.  However, such a reorganization would
+  require an amount of proof of work roughly equal to the annual output
+  of all active Bitcoin miners (at the time of writing), so this is
+  considered to be both very unlikely and indicative of a threat that
+  could prevent consensus formation anyway.  The hardcoding, which is
+  similar to the [BIP90][] hardcoding made a couple years ago,
+  simplifies Bitcoin Core's consensus code.  For more information, see
+  the [mailing list post][buried post] or [PR #16060][Bitcoin Core
+  #16060].
+
+- **New Optech Compatibility Matrix:** a [new feature][compatibility
+  matrix] on the Optech
+  website shows what wallets and services support certain recommended
+  features, currently opt-in Replace-by-Fee (RBF) and segwit (with more
+  comparisons planned for the future).  The matrix is designed to help
+  developers gauge how well supported features are and learn from the
+  designs of early adopters.  For details, please see our [announcement
+  post][compat announce].
+
+## Bech32 sending support
+
+*Week 23 of 24 in a [series][bech32 series] about allowing the people
+you pay to access all of segwit's benefits.*
+
+The *News* section from this week's newsletter introduced a new feature
+on the Optech website.  {% include specials/bech32/23-compat.md %}
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #16248][] extends the whitelisting configuration options
+  in Bitcoin Core to allow specifying which services should be provided
+  to different IP addresses or ranges.  For example, a motivation for
+  the change was allowing bloom filters to be provided to particular
+  peers (such as a user's own lightweight wallet) even if the filters
+  are disabled by default.  This can be done with
+  `-whitelist=bloomfilter@1.2.3.4/32`.  If only an IP
+  address is provided (i.e., no permissions are specified), the behavior
+  is the same as before.
+
+- [Bitcoin Core #16383][] changes RPC commands that accept an
+  `include_watchonly` parameter so that it is automatically set to True
+  for wallets that have private keys disabled (i.e. that are only useful
+  as watch-only wallets).  This ensures results for watch-only addresses
+  are included in results.
+
+- [Bitcoin Core #15986][] extends the `getdescriptorinfo` RPC with an
+  additional `checksum` field that returns the checksum for the
+  descriptor provided by the user.  This differs from the checksum
```

Comment:
I don't think "This differs from the checksum described in Newsletter #34" is the clearest way to introduce this. Suggested alternative:

"Bitcoin Core #15986 extends the getdescriptorinfo RPC described in Newsletter #34. getdescriptorinfo returns the _normalized_ descriptor (which removes private keys and makes other modifications) along with its checksum. It now also returns a `checksum` field which gives the checksum of the _unnormalized_ descriptor provided by the user. Descriptor checksums follow a # character at the end of the string as described in the output script descriptor documentation."